proto/workflow.proto

```diff
@@ -120,6 +122,71 @@ message GetWorkflowsResponse {
   repeated Workflow workflow = 2;
 }
 
+message ActionHistory {
+  // The name of the action spawned by the workflow--this is stored in the
+  // 'pattern' field of the invocation.
+  string action_name = 1;
+
+  // Information about a single run of this action.
+  message ActionHistoryEntry {
```

Comment:
I'm not sure if the precedent is 100% there, but wdyt about removing the redundant prefix here (and below) since these are nested inside of ActionHistory?

Would make the callsites less stuttery: `ActionHistory_ActionHistoryEntry` -> `ActionHistory_Entry`

Reply:
sure, done